### PR TITLE
add a fallback for zapper list

### PR DESCRIPTION
**Pull request by andytcf**

### CODE REVIEW

File: queries/tokenLists/useZapperTokenList.ts [hunk 1]
```diff
@@ -24,7 +24,7 @@ const useZapperTokenList = (options?: QueryConfig<TokenListQueryResponse>) => {
 	return useQuery<TokenListQueryResponse>(
 		QUERY_KEYS.TokenLists.Zapper,
 		async () => {
-			const response = await axios.get<TokenListResponse>(EXTERNAL_LINKS.TokenLists.Zapper);
+			const response = await axios.get<TokenListResponse>(EXTERNAL_LINKS.TokenLists.OneInch);
```
Comment (vbstreetz) [inline queries/tokenLists/useZapperTokenList.ts:27]: Should this remain as is?

**Review by vbstreetz — CHANGES_REQUESTED** [review]
Added a comment but looking good.

File: queries/tokenLists/useZapperTokenList.ts (hunk 1 above)
Comment (andytcf) [reply queries/tokenLists/useZapperTokenList.ts:27]: ah yes it should

**Review by vbstreetz — APPROVED** [review]
looks good 👍🏽

**Pull request merged**